clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/SimpleClusterTierClientEntity.java

```diff
@@ -288,4 +300,26 @@ private Duration getTimeoutDuration(EhcacheOperationMessage message) {
       }
     }
   }
+
+  private static void waitForLatch(long nanos, CountDownLatch latch) {
```

Comment:
Can you rename that to waitForInitCompletion or something like that. Why do you need to pass the latch in parameter? Or maybe this method should go in some util class.

Reply:
just followed an existing pattern in this class. See waitFor..Ideally I should just reuse waitFor and pass in a consumer.. will do that..

Reply:
btw, used a wait/notify instead and changed method name to waitForInitCompletion as per your suggestion.